### PR TITLE
feat: station info example

### PR DESCRIPTION
**Pull request by Khrystyna**
In this example, we query station data and suggest how you could parse it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,10 +21,11 @@ Available examples:
 1. Build a simple route [example](https://chargetrip.github.io/examples/route/);
 2. Select a car [example](https://chargetrip.github.io/examples/car/);
 3. Show stations on a map [example](https://chargetrip.github.io/examples/stations/);
-4. Show stations using the Vector Tile Server [example](https://chargetrip.github.io/examples/tile-server/);
-5. Build a route and display an elevation profile [example](https://chargetrip.github.io/examples/elevation-plot/).
-6. Build a route and recalculate if your SOC changes [example](https://chargetrip.github.io/examples/state-of-charge/)
-7. Build a route and display stations along the route [example](https://chargetrip.github.io/examples/stations-along-route/);
+4. Query specific station [example](https://chargetrip.github.io/examples/station-info/);
```
Comment (lsmienk) [inline README.md:24]: I would say **query a specific station**

File: public/index.html [hunk 2]
```diff
@@ -98,6 +98,10 @@ <h2>Select a car 🚘</h2>
           <h2>Query stations 🗺</h2>
           <p>Example of how to query the 20 closest stations and show them on a map</p>
         </a>
+        <a href="station-info/" class="example">
```
Comment (lsmienk) [inline public/index.html:101]: same here 'a station'

File: station-info/README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+To see our Chargetrip API in action, you can go to the [Playground](https://playground.chargetrip.com/). It has a big collection of mutations/queries for you to experience the power of our API.
+
+This example is built with vanilla JS. To establish a connection with Chargetrip API, we use [urql](https://formidable.com/open-source/urql/) - lightweight GraphQL client.
+We use our Playground environment for this example. Our Playground has a station database that is populated with freely available European station data from [OCM](https://openchargemap.org/site). It means that only part of our extensive database is available. You need a registered `x-client-id` to access the full database. Importing your own database or using one of the databases Chargetrip has an integration with, is possible. For more details, please contact us.
+
+### Steps to take
+
+To get stations around some location and fetch specific station details:
```
Comment (lsmienk) [inline station-info/README.md:17]: I would not use around some location but around a location or maybe even around a specific location.

**Review by lsmienk — CHANGES_REQUESTED** [review]
Looks really good! I really like how you use the utils to get the names you want to display for all possible connectors etc. Just some small spelling changes I would recommend.

File: station-info/station.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+  const connectors = station.chargers?.map(charger => {
+    const status = getConnectorStatus(charger);
+    return {
+      name: getConnectorName(charger.standard),
+      icon: getConnectorIcon(charger.standard),
+      power: charger.power,
+      status,
+      standard: charger.standard,
+      availabilityInfo: `${status === ConnectorStatus.UNKNOWN ? '-' : charger.status[status]}/${charger.total}`,
+      availabilityLabel: gerConnectorStatusLabel(status),
```
Comment (timkuilman) [inline station-info/station.js:39]: typo: getConnectorStatusLabel

**Review by timkuilman — CHANGES_REQUESTED** [review]
Just one typo but LGTM!

File: README.md (hunk 1 above)
Comment (Khrystyna) [reply README.md:24]: 👍

File: station-info/README.md (hunk 3 above)
Comment (Khrystyna) [reply station-info/README.md:17]: 👍

File: station-info/station.js (hunk 4 above)
Comment (Khrystyna) [reply station-info/station.js:39]: 👍

**Review by timkuilman — APPROVED** [review]

File: public/index.html (hunk 2 above)
Comment (Khrystyna) [reply public/index.html:101]: 👍

**Review by lsmienk — APPROVED** [review]

**Pull request merged**